### PR TITLE
Fix/report auto generation timezone

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ZH Card](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/bcgov/cas-cif/1136)

### CODE REVIEW

**Review by pbastia — DISMISSED** [review]
very nice

**Comment by Sepehr-Sobhani** [comment]
> very nice

@pbastia Unfortunately this fix was not enough. still looking for a solution.

File: schema/deploy/mutations/generate_reports@1.0.0-rc.5.sql [hunk 1]
```diff
@@ -0,0 +1,68 @@
+-- Deploy cif:mutations/generate_reports to pg
+
+begin;
+
+create or replace function cif.generate_reports(revision_id int, report_type text, start_date timestamptz, end_date timestamptz)
+  returns setof cif.form_change as $generate_reports$
```
Comment (pbastia) [inline schema/deploy/mutations/generate_reports@1.0.0-rc.5.sql:6]: Just a line break for consistency with how we've written those in the past

```suggestion
returns setof cif.form_change as 
$generate_reports$
```

**Review by pbastia — CHANGES_REQUESTED** [review]
Awesome! Just a style comment

**Review by pbastia — APPROVED** [review]

**Pull request merged**